src/flows/shared/steps/OrganizeCollection/Sidebar/SidebarNftIcon.tsx

```diff
@@ -69,7 +69,7 @@ export const StyledSidebarNftIcon = styled.div`
   width: 64px;
   height: 64px;
   overflow: hidden;
-  margin: 6px;
```

Comment:
sometimes the sidebar nft icons were getting squished to 2 columns, so this gives a bit of breathing room so that there's enough space for 3 columns in the sidebar